lib/wrap-guide-element.coffee

```diff
@@ -78,6 +79,7 @@ class WrapGuideElement extends HTMLDivElement
     column = @getGuideColumn(@editor.getPath(), @editor.getGrammar().scopeName)
     if column > 0 and @isEnabled()
       columnWidth = @editorElement.getDefaultCharacterWidth() * column
+      columnWidth -= @editor.getScrollLeft()
       @style.left = "#{columnWidth}px"
```

Comment:
I'd be fine with that so long as it doesn't hurt our compositing performance. The Chromium compositor seems great now so it probably won't.